pkg/plugins/policies/meshmetric/plugin/v1alpha1/plugin_test.go

```diff
@@ -268,5 +268,46 @@ var _ = Describe("MeshMetric", func() {
 				).
 				Build(),
 		}),
+		Entry("multiple_otel", testCase{
```

Comment:
Can this be done via golden file? it's really hard to read.

Reply:
probably, but we never do it via golden file in policies test. I think adding this won't be trivial and I think that there are more important things to take care of